src/sentry/tasks/on_demand_metrics.py

```diff
@@ -39,6 +39,7 @@
 
 # TTL for cardinality check
 _WIDGET_QUERY_CARDINALITY_TTL = 3600 * 24 * 7  # Cardinality outcome is valid for 7 days.
+_COLUMN_CARDINALITY_TTL = 3600 * 24  # Cardinality outcome is valid for 1 days.
```

Comment:
Maybe we should make this explicitly the "save/edit" widget TTL and have a different TTL for the task? 24 hours is a long time for fields that can change when they are just introduced and someone is adding a widget.

Reply:
from slack; Going to change the column TTL to 1hr, and the query period for widget saves from 30m to 1hr as well